app/models/course.rb

```diff
@@ -1,12 +1,22 @@
 class Course < ApplicationRecord
 
-  has_many :sections
-  has_many :course_dependents
-  has_many :dependents, through: :course_dependents
-  has_many :course_requirements
-  has_many :requirements, through: :course_requirements
-  has_many :course_semesters
-  has_many :semesters, through: :course_semesters
-  validates_presence_of :name, :department, :units, :description, :mini
+	after_initialize :set_defaults
+  
+	has_many :sections
+	has_many :course_dependents
+	has_many :dependents, through: :course_dependents
+	has_many :course_requirements
+	has_many :requirements, through: :course_requirements
+	has_many :course_semesters
+	has_many :semesters, through: :course_semesters
+	validates_presence_of :code, :name, :mini
+
+	def set_defaults
+    	# Only set if time_zone IS NOT set
```

Comment:
Why does timezone matter?

Reply:
It doesn't! leftover comment from copy/pasted code